frontend/src/config.ts

```diff
@@ -0,0 +1,5 @@
+const config: { BASE_URL: string } = {
```

Comment:
This is not an environment variable..
I expect an axios config, and then the config takes in an environment variable as the baseUrl